tcms/testruns/static/testruns/js/get.js

```diff
@@ -180,13 +180,36 @@ function renderTestExecutions(testExecutions) {
     })
 
     treeViewBind();
+    $('.test-executions-count').html(testExecutions.length)
     renderAdditionalInformation(testExecutions, testCaseIds)
 }
 
 function renderAdditionalInformation(testExecutions, testExecutionCaseIds) {
-    $('.test-executions-count').html(testExecutions.length);
```

Comment:
Why move this line out of the function ?

Reply:
I reorganized the functions. Before we reused `renderTestCaseInformation` but `renderAdditionalInformation` was called only once. NOw, I added the history, which is not TC information, but we need to call it every time we rerender the row. So I made `renderAdditionalInformation` reusable by removing this file from there (because we only want that in the first load of the page, not on rerendering a row)

Reply:
`history, which is not TC information`

You need better naming of things:

-  there are no test cases here, but test executions so function names should reflect that
- there is TE row information (which you call Test Case information) and TE additional information
- ^^^ these two are mixed ATM and should be separated and refactored very soon b/c this page will die once somebody creates a test run with over a 1000 executions in it.